commit 7b95fc9db45ef70dc12097faee264670dbb0da07
Author: Radek Novacek <rnovacek@redhat.com>
Date:   Thu Jan 28 13:51:46 2016 +0100

    Documentation and man page fixes
    
    * Remove requirement for AllowUnencrpyted from hyperv readme.
    * Note that hypervisor_id=hwuuid is not supported on hyperv

diff --git a/README.hyperv b/README.hyperv
index 3aba0d1..9207c50 100644
--- a/README.hyperv
+++ b/README.hyperv
@@ -14,20 +14,13 @@ In order to make virt-who connection to Hyper-V work, some steps needs to be don
         netsh advfirewall firewall set rule group="Remote Administration" new enable=yes
 
 
-3. Unencrypted connection must be enabled for HTTP (not required for HTTPS)
-
-    Following command can be used on Hyper-V server:
-
-        winrm set winrm/config/service @{AllowUnencrypted="true"}
-
-
-4. Only Basic and NTLM authentication methods are supported
+3. Only Basic and NTLM authentication methods are supported
 
     Verify that at least one of methods Basic or Negotiate is enabled (True)
 
         winrm get winrm/config/service/auth
 
-5. Use "Administrator" account. If for security concerns "Administrator" account
+4. Use "Administrator" account. If for security concerns "Administrator" account
     cannot be used to authenticate to Hyper-V, then it is necessary to create
     an additional user and assign the user to the following groups
     (Windows 2012 Server): "Hyper-V Administrators" and "Remote Management Users".
diff --git a/virt-who-config.5 b/virt-who-config.5
index 002595d..4f4ac3b 100644
--- a/virt-who-config.5
+++ b/virt-who-config.5
@@ -122,7 +122,7 @@ Only hosts which uuid (or hostname or hwuuid, based on \fBhypervisor_id\fR) is s
 Hosts which uuid (or hostname or hwuuid, based on \fBhypervisor_id\fR) is specified in comma-separated list in this option will \fBNOT\fR be reported. Put the value into the double-quotes if it contains special characters (like comma).
 .TP
 \fBhypervisor_id\fR
-Property that should be used as identification of the hypervisor. Can be one of following: \fBuuid\fR, \fBhostname\fR, \fBhwuuid\fR. Note that some virtualization backends don't have all of them implemented. Default is \fBuuid\fR. Applicable to esx, hyperv and rhevm only. This property is meant to be set up before initial run of virt-who. Changing it later will result in duplicated entries in the subscription manager.
+Property that should be used as identification of the hypervisor. Can be one of following: \fBuuid\fR, \fBhostname\fR, \fBhwuuid\fR. Note that some virtualization backends don't have all of them implemented. Default is \fBuuid\fR. Applicable to esx and rhevm only. This property is meant to be set up before initial run of virt-who. Changing it later will result in duplicated entries in the subscription manager.
 
 .SH EXAMPLE
 [test-esx]
